src/citrine/resources/gemtables.py

```diff
@@ -41,12 +41,35 @@ class GemTable(Resource['Table']):
     """:str: URL pointing to the location of the GEM Table's contents.
     This is an expiring download link and is not unique."""
 
+    _config = properties.Optional(properties.Object(TableConfig), "config")
```

Comment:
By defining these as optional properties, we'll seamlessly start using them if they're added to the API responses.